frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -236,6 +221,21 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAG_OPENSHIFT_PIPELINE],
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      perspective: 'dev',
+      group: 'resources',
+      componentProps: {
+        name: 'Application Stages',
+        href: '/applicationstages',
+        testID: 'application-stages-header',
+      },
+    },
+    flags: {
+      required: [FLAG_OPENSHIFT_GITOPS],
```

Comment:
Why not rename the flag as well 
```suggestion
      required: [FLAG_OPENSHIFT_APPLICATION_STAGES],
```

Reply:
The operator is still GitOps and the flag can also be used at multiple places and not just for `Application Stages` page.